src/Core/Geometry/PolyLine.inl

```diff
@@ -1,4 +1,5 @@
 #include <Core/Geometry/PolyLine.hpp>
+#include <Core/Math/LinearAlgebra.hpp>
```

Comment:
```suggestion
```

Reply:
I'm checking, but Math.hpp is needed here ...

Reply:
Ok, I let you find out ;)

Reply:
oh cotan is from LinearAlgebra, so it's needed here